ui/src/containers/InventoryLayout.js

```diff
@@ -42,19 +43,44 @@ const headCells = [
   { id: 'unitOfMeasurement', align: 'right', disablePadding: false, label: 'Unit of Measurement' },
   { id: 'bestBeforeDate', align: 'right', disablePadding: false, label: 'Best Before Date' },
 ]
+ 
 
 const InventoryLayout = (props) => {
+  
   const classes = useStyles()
   const dispatch = useDispatch()
   const inventory = useSelector(state => state.inventory.all)
```

Comment:
You can retrieve products similar to inventory here:
`const products = useSelector(state => state.products.all)`